mllib-tests/v1p4/src/main/scala/mllib/perf/MLAlgorithmTests.scala

```diff
@@ -391,14 +392,21 @@ class NaiveBayesTest(sc: SparkContext)
 
     val threshold: Double = doubleOptionValue(THRESHOLD)
     val sf: Double = doubleOptionValue(SCALE)
+    val modelType = stringOptionValue(MODEL_TYPE)
```

Comment:
Please double check. I think this could throw an exception if `MODEL_TYPE` is not set, which is the case in multinomial naive Bayes.